yarn.lock

```diff
@@ -526,19 +524,6 @@ angular2react@^2.0.0:
     react "^15.4.2"
     react-dom "^15.4.2"
 
-angular2react@^2.0.3:
```

Comment:
Yeah... it's weird that angular2react@^2.0.3 was ever in here, because we have ^2.0.0 in package.json. Maybe just leftover from an old thing.